src/components/WelcomeToSDC.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { useSession } from "next-auth/client";
+import {
+    Box,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Image,
+    Button,
+    Flex,
+} from "@chakra-ui/react";
+import { Tabs, TabList, TabPanels, Tab, TabPanel } from "@chakra-ui/react";
+
+export const WelcomeToSDC: React.FC = () => {
+    const goToLogin = () => {
+        window.location.href = "/login";
+    };
+
+    // TODO remove test data and get new images
+    const title = "Welcome to SDC";
+    const desc1 = "Registration for Summer 2021 classes begins June 31, 2021!";
+    const text1 =
+        "Browse through programs below and then select the day/time preferred! We make sure that 100% of our participants' demonstrated needs are met.";
+    const text2 =
+        "For volunteers, create an account to volunteer for SDC and then select the program you want to volunteer for!";
+    const img1 =
+        "https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/active_kids_other/1800x1200_active_kids_other_alt.jpg";
+    const img2 =
+        "https://www.verywellfamily.com/thmb/dIuXfSzEeILbXa3aSSLuU1xvFR8=/2121x1414/filters:fill(D7DFF5,1)/children-running-in-park-537632931-5c49f59d46e0fb00016e2ad6.jpg";
+    const [session, loading] = useSession();
+    return (
+        <Tabs>
+            <TabList>
+                <Tab color="#0C53A0">Parents</Tab>
```

Comment:
There's a blue highlight around the tab which lingers until you click somewhere else on the screen, and when you click the tab it goes grey for like a split second - I think it might better to get rid of both?? 
<img width="322" alt="Screen Shot 2021-07-24 at 10 17 38 PM" src="https://user-images.githubusercontent.com/42329517/126885486-86e5ed50-f9d6-491b-9cf2-b93bb0250733.png">

```suggestion
                <Tab _focus={{}} _active={{}} color="#0C53A0">Parents</Tab>
```

Reply:
damn thats big brain :O